.github/ISSUE_TEMPLATE/bug_report_form.yml

```diff
@@ -0,0 +1,169 @@
+name: Bug Report
+description: File a bug report
+title: "[BUG] "
+labels:
+  - kind/possible bug
+  - meta/needs triage
+  - priority/medium
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for participating in this project and reporting a bug. Docker Mail Server (DMS) is a community-driven project, and each contribution counts.
+
+        Please **fill out all the fields and checkboxes of this form** to make it easier for maintainers to understand the problem and to solve it. If this template is not adhered to, it will be marked with a label called `meta/no template - no support` and will not be worked on.
+
+        **Make sure** you read through the whole README, the documentation and the issue tracker before opening a new bug report.
+  - type: input
+    id: affected-components
+    attributes:
+      label: Affected Component(s)
+      description: What is affected by this bug? Please describe it with only a few words here. Detailed description can be given later.
+      placeholder: Postfix is not starting
+    validations:
+      required: true
+  - type: textarea
+    id: when-does-it-occur
+    attributes:
+      label:  When does this occur and what happened?
+      description: Tell us what happened. Use Markdown formatting for code!
+      placeholder: Although `DMS_DEBUG=1` is set, I see no debug.
+      render: Markdown
+    validations:
+      required: true
+  - type: textarea
+    id: what-did-you-expect-to-happen
+    attributes:
+      label:  What did you expect to happen?
+      description: Tell us what you expected.
+      placeholder: I expected to see debug messages.
+      render: Markdown
+    validations:
+      required: true
+  - type: textarea
+    id: how-do-we-replicate-this-issue
+    attributes:
+      label: How do we replicate the issue?
+      description: What did you do and how can we replicate this issue=
+      value: |
+        1.
+        2.
+        3.
+        ...
+      render: Markdown
+    validations:
+      required: true
+  - type: input
+    id: mailserver-version
+    attributes:
+      label: Mail server version
+      description: What version are you running? Paste your image tag here.
+      placeholder: v10.1.2
+    validations:
+      required: true
+  - type: dropdown
+    id: main-memory
+    attributes:
+      label: How much RAM is available?
```

Comment:
CPU count could also be important

Reply:
Good idea! I will add this later.

Reply:
Done. [e556f64932f6ce98016cac6763f218a4ff01f97c]